app/src/main/shared/styles/variables.scss

```diff
@@ -0,0 +1,828 @@
+// Variables
```

Comment:
This file is copied from bootstrap source files. I have changed the table-border-color var to a darker grey...